README.md

```diff
@@ -63,121 +63,44 @@ ONNX gives the flexibility to serve your model in a framework-agnostic environme
 Support includes [PyTorch,](https://pytorch.org/docs/stable/onnx.html) [TensorFlow,](https://github.com/onnx/tensorflow-onnx) [Keras,](https://github.com/onnx/keras-onnx) and [many other frameworks](https://github.com/onnx/onnxmltools).
 
 The DeepSparse Engine is available in two editions: 
-1. **The Community Edition** is open-source and free for evaluation, research, and non-production use with our [Engine Community License](https://neuralmagic.com/legal/engine-license-agreement/).
-2. **The Enterprise Edition** requires a Trial License or [can be fully licensed](https://neuralmagic.com/legal/master-software-license-and-service-agreement/) for production, commercial applications. 
-
-## Installation Overview
-
-- [Hardware Support and System Requirements](https://docs.neuralmagic.com/deepsparse/source/hardware.html)
-- [DeepSparse Community Edition](https://github.com/neuralmagic/deepsparse/main/README.md#deepsparse-community-edition)
-- [DeepSparse Enterprise Edition](https://github.com/neuralmagic/deepsparse/main/README.md#deepsparse-enterprise-edition)
+1. [**The Community Edition**](#community-edition-installation) is open-source and free for evaluation, research, and non-production use with our [Engine Community License](https://neuralmagic.com/legal/engine-license-agreement/).
+2. [**The Enterprise Edition**](#enterprise-edition) requires a Trial License or [can be fully licensed](https://neuralmagic.com/legal/master-software-license-and-service-agreement/) for production, commercial applications.
 
 ## Features
 
 - 🔌 [DeepSparse Server](https://github.com/neuralmagic/deepsparse/tree/main/src/deepsparse/server)
 - 📜 [DeepSparse Benchmark](https://github.com/neuralmagic/deepsparse/tree/main/src/deepsparse/benchmark)
 - 👩‍💻 [NLP and Computer Vision Tasks Supported](https://github.com/neuralmagic/deepsparse/tree/main/examples)
 
-## Installation Guide
-
-### 🧰 Hardware Support and System Requirements
+## 🧰 Hardware Support and System Requirements
 
 Review [CPU Hardware Support for Various Architectures](https://docs.neuralmagic.com/deepsparse/source/hardware.html) to understand system requirements. 
 The DeepSparse Engine works natively on Linux; Mac and Windows require running Linux in a Docker or virtual machine; it will not run natively on those operating systems.
 
 The DeepSparse Engine is tested on Python 3.7-3.10, ONNX 1.5.0-1.12.0, ONNX opset version 11+, and manylinux compliant. 
 Using a [virtual environment](https://docs.python.org/3/library/venv.html) is highly recommended. 
 
-### DeepSparse Community Edition
+## Community Edition Installation
 
-Install the Community Edition as follows: 
```

Comment:
If we keep the header as DeepSparse Community Edition then put the installation section under so it's the same as the enterprise section. and make that say just ###Installation. 

So below, let's change it to DeepSparse Enterprise Edition instead of Enterprise Edition and do the ### Installation header there too. 

Add ###Installation

Reply:
Resolved based on new direction of having enterprise live on docs.neuralmagic.com